src/content/sessions/2024-01-19.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Glutinum"
+preview: "Glutinum"
+isDraft: true
+date: 2024-01-19T14:00:00.000Z
+slug: "2024/01/19"
+champion: "Maxime Mangel"
+zoomLink: "https://us06web.zoom.us/j/83166916355?pwd=qWtIEIFmXvd3FDaTYRM8pRDKm3TAmU.1"
+zoomPasscode: "glue"
+issueLink: "https://github.com/glutinum-org/cli/issues/24"
```

Comment:
@nojaf The exact issue link is probably: https://github.com/glutinum-org/cli/issues/28

This is the issue focused on a single the SyntaxKind (MethodSignature) that we will be working on. It is marked as closed because I merged our dry run session in main.

I let it up to you to decide if we should update the link or not. It doesn't change much

Reply:
Oh, I'll leave it. It is the original issue that the session was asked for in https://github.com/amplifying-fsharp/sessions/issues/34.
It also doesn't matter that much.